app/models/header_lookup.rb

```diff
@@ -61,24 +63,31 @@ def pid_lookup_by_field(term="", field="")
         end
       end
 
-      memoized_terms[term] = term_pid
-      File.write(memoized_file, memoized_terms)
+      @@all_memoized_terms[memoized_vocab][term] = term_pid
+      File.write(memoized_file, @@all_memoized_terms[memoized_vocab])
```

Comment:
This should be some form of `JSON.dump`, right? (otherwise we are just dumping the hash again?)

Reply:
Oh jeez, good catch.

Reply:
Thankfully this was as simple as tacking a `.to_json` on